lib/batch_type.js

```diff
@@ -28,27 +28,28 @@ const batchType = as.batchTypes
 // ========================================================================
 // Constants
 // ========================================================================
-
-/**
- * An instance of class Record that is used in a batch for read operations.
- * @const {number}
- */
-exports.BATCH_READ = batchType.BATCH_READ
-
-/**
- * An instance of class Record that is used in a batch for write operations.
- * @const {number}
- */
-exports.BATCH_WRITE = batchType.BATCH_WRITE
-
-/**
- * An instance of class Record that is used in a batch for applying record.
- * @const {number}
- */
-exports.BATCH_APPLY = batchType.BATCH_APPLY
-
-/**
- * An instance of class Record that is used in a batch for removal operations.
- * @const {number}
- */
-exports.BATCH_REMOVE = batchType.BATCH_REMOVE
+exports.types = {
```

Comment:
As discussed, maybe try out `exports = { ... }` to see if you can make this work with TypeScript without breaking the user-facing interface.

Reply:
Updated.